Schronisko_app.py

```diff
@@ -0,0 +1,129 @@
+from flask import Flask, request, render_template, g, flash, redirect, url_for, abort
```

Comment:
not a critical issue but it is recommended to use snake  case and stick to English, so `app.py` or `shelter_app.py`